tmt/schemas/provision/beaker.yaml

```diff
@@ -49,5 +49,8 @@ properties:
   beaker-job-owner:
     type: string
 
+  job-group:
```

Comment:
`beaker-job-group`, see `beaker-job-owner` just above.

Reply:
sure,and updated:https://github.com/teemtee/tmt/pull/3397/commits/3d5cd136b12323e36e5772d758ef25552e22553e ^^